sphinx_doc/howtos/tutorials/docs/navigation2_on_real_turtlebot3.rst

```diff
@@ -0,0 +1,161 @@
+.. _navigation2-on-real-turtlebot3:
+
+Navigation2 on a Real Turtlebot 3
+*********************************
+
+- `Overview`_
+- `Requirements`_
+- `Tutorial Steps`_
+- `Videos`_
+
+Overview
+========
+
+This tutorial shows how to control and navigate Turtlebot 3 using the ROS2 Navigation2 on a real Turtlebot 3 robot.
+Before completing this tutorials, completing the :ref:`navigation2-with-turtlebot3-in-gazebo`. is highly recommended especially if you are new to ROS and Navigation2.
+
+Turtlebot is a low-cost, personal robot kit with open-source software (ROS2).
+Turtblebot robots are widely supported by the ROS community.
+You can find many applications and examples of Turtlebot projects on the Internet.
+You can find more information about Turtlebot3 `here. <http://emanual.robotis.com/docs/en/platform/turtlebot3/overview/>`_
+
+Links to the robot kits and Github repo.
+
+- `Turtlebot Burger <http://www.robotis.us/turtlebot-3-burger-us/>`_
+- `Turtlebot Waffle <http://www.robotis.us/turtlebot-3-waffle-pi/>`_
+- `Turtlebot 3 Github <https://github.com/ROBOTIS-GIT/turtlebot3>`_
+
+This tutorial consists of two parts. In the first part, you learned how to use Turtlebot 3 robots in simulation (Gazebo).
+In this part, you will learn how to control a real Turtlebot Waffle using Navigation2.
+
+``ROS2 Dashing`` and ``Navigation2 Dashing 0.2.4`` are used to create this tutorial.
+You should be able to do this tutorial using other Navigation2 versions as well.
+
+This tutorial may take about 1-2 hours to complete. 
+It depends on your experience with ROS, robots, and what computer system you have.
+
+Requirements
+============
+
+- `Install ROS2 <https://index.ros.org/doc/ros2/Installation/>`_
+
+- Install Navigation2
+
+    - ``sudo apt install ros-<ros2-distro>-navigation2 ros-<ros2-distro>-nav2-bringup``
+
+- Install Turtlebot 3
+
+    - ``sudo apt install ros-<ros2-distro>-turtlebot3*``
+
+- `Setup Turtlebot 3 <http://emanual.robotis.com/docs/en/platform/turtlebot3/ros2/#setup>`_
+
+Tutorial Steps
+==============
+
+0- Setup Your Enviroment Variables
+----------------------------------
+
+Run the following commands first whenever you open a new terminal during this tutorial.
+
+- ``source /opt/ros/<ros2-distro>/setup.bash``
+- ``export TURTLEBOT3_MODEL=waffle``
+
+1- Launch Turtlebot 3 Robot State Publisher
+-------------------------------------------
+
+Launch Turtlebot 3 robot state publisher,
+
+  ``ros2 launch turtlebot3_bringup turtlebot3_state_publisher.launch.py use_sim_time:=False``
+
+Make sure use_sim_time is set to **False**.
+
+2- Launch Navigation2
+---------------------
+
+You need to have a map of the environment where you want to Navigate Turtlebot 3. Required files:
```

Comment:
`You need to have a map of the environment where you want to Navigate Turtlebot 3, or create one live with SLAM`

Just listing the slam thing here passively will help people thinking "how do I make a map if I cant move?!" if they can make one while doing this if they're knowledgeable enough to know about SLAM. If not, they'll just pass over it.

Reply:
+ I added a link to the SLAM use case tutorial as well. It hasn't been converted to .rst format yet, so the link open the .md file in the repo.

Reply:
nice!